### PR TITLE
Overhaul website content for proc resource

### PR DESCRIPTION
**Pull request by safrazier17**
Reordered lots of content for better legibility. Reduced overall content substantially. This was for a few reasons, but primarily because:
* Much of the copy dated to Q4 2021 and was out of date or no longer relevant
* The copy that was here was mostly copied from the white paper without much attention paid to organization or easy navigability on the web
* We are hoping to direct more traffic to the site with the imminent launch of our procurement resource so it seemed like a good time for a clean up

### CODE REVIEW

File: docs/vision.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Discover and make informed choices about their mobility options in scheduled and/or real-time by utilizing the highest possible quality of information with the least possible barriers to accessing that information;  
+- Maintain their privacy and security; and  
+- Travel with the minimum possible friction arising from accessing data related to their mode(s) of travel.  
+
+**[Mobility service providers](definitions.md#mobility_service_provider)** can:  
+
+- Interface with other mobility providers to meet riders’ needs;  
+- Use the best available data to support analytics and real-time decision-making for their planning, scheduling, and operating needs;  
+- Select the best available technology and equipment to provide their service without vendor lock-in; and  
+- Easily use their data and data from other [mobility service providers](definitions.md#mobility_service_provider) to increase rider satisfaction and the customer experience through service efficiency and service quality.
```
Comment (matikin9) [inline docs/vision.md:16]: sounds like we're missing a word?

to increase rider satisfaction and `improve` the customer experience through service efficiency and service quality.

Comment (matikin9) [reply docs/vision.md:16]: otherwise, looks good to me!

**Comment by tsherlockcraig** [comment]
These changes lgtm. I'm a little hesitant on the removal of the process page but ultimately I think taking it out is right, until we have the staff to commit to substantive ongoing updates and stick to what's committed to. The history information about launch is no longer needed.

**Review by thekaveman — APPROVED** [review]
Applied some minor formatting cleanup to make the linter happy.

I also applied the same fix as we did for the ODS website a while back, to integrate each page's Table of Contents (TOC) into the normal page menu on the left, instead of separating it out to the right:

**Before**

image

**After**

image

File: docs/vision.md (hunk 1 above)
Comment (safrazier17) [reply docs/vision.md]: thanks @matikin9 !

**Pull request merged**